src/Resources/Resources/help/Stop-AzDeployment.md

```diff
@@ -42,14 +62,35 @@ This cmdlet stops only one running deployment. Use the *Name* parameter to stop
 
 ### Example 1
 ```
-PS C:\>Stop-AzDeployment -Name "deployment01"
+PS C:\>Stop-AzDeployment -ScopeType "Subscription" -Name "deployment01"
```

Comment:
@Tiano2017 same comment about adding the corresponding output to all examples in this file (if this cmdlet does return anything)